plugins/ctf.rb

```diff
@@ -89,6 +89,8 @@ def list_ctfs(ctfs, message)
       if @credentials.has_key?(ctf.title) && @creds_authorized.include?(message.user.nick)
         creds = @credentials[ctf.title].reject(&:nil?).map { |x| "'#{x}'" }.join(':')
         msg << " - #{creds}"
+      elsif @credentials.has_key?(ctf.title)
+        msg << " - active/planned"
```

Comment:
I think I would rather have this say "participating". Whether it's planned can be inferred from the section the CTF appears in. I also don't think "active" conveys that there are people participating in the given event.